ios/RNMagicScript/components/UiNodeSelector.swift

```diff
@@ -65,4 +73,15 @@ class UiNodeSelector: NodeSelecting  {
 
         return node as? Dragging
     }
+    
+    func focusInterceptedNodes() -> [BaseNode] {
+        var result: [BaseNode] = []
+        rootNode.enumerateBaseNodes { node in
```

Comment:
the enumeratedBaseNodes are going to be in the correct order for a hit test?

Reply:
That's a good question. I was wondering if we should sort the result nodes by the distance to the camera. But in theory, we should not allow activating multiple components that can intercept focus (= it's not possible to open multiple dropdown lists - theoretically).

Reply:
code updated